Docs/user_guide/settings.md

```diff
@@ -142,3 +142,25 @@ The following environment variables can be set before the application is started
 - `SLICER_EXTENSIONS_MANAGER_SERVER_URL`: URL of the extensions manager backend with the `/api` path. Default value is retrieved from the settings using the key `Extensions/ServerUrl`.
 - `SLICER_EXTENSIONS_MANAGER_FRONTEND_SERVER_URL`: URL of the extension manager frontend displaying the web page. Default value is retrieved from the settings using the key `Extensions/FrontendServerUrl`.
 - `SLICER_EXTENSIONS_MANAGER_SERVER_API`: Supported values are `Midas_v1` and `Girder_v1`. Default value is hard-coded to `Girder_v1`.
+
+### Qt built-in command-line options
+
+Slicer application accepts standard Qt command-line arguments that specify how Qt interacts with the windowing system.
+
+Examples of options:
+- `-qwindowgeometry geometry`, specifies window geometry for the main window using the X11-syntax. For example: `-qwindowgeometry 100x100+50+50`.
+- `-display hostname:screen_number`, switches displays on X11 and overrides the `DISPLAY` environment variable.
+- `-platform windows:dpiawareness=[0|1|2]`, sets the [DPI awareness](https://doc.qt.io/qt-5/highdpi.html#microsoft-windows) on Windows.
```

Comment:
I will have to confirm that this syntax works as expected. Will get back to this a few hours ...

Reply:
> I will have to confirm that this syntax works as expected. Will get back to this a few hours ...

:heavy_check_mark: See https://github.com/commontk/AppLauncher/pull/126

Reply:
These are good examples. Hopefully they work!